tests/integration/ceph_base_deploy_test.go

```diff
@@ -45,6 +45,7 @@ const (
 	helmMinimalTestVersion            = "1.17.0"
 	upgradeMinimalTestVersion         = "1.18.0"
 	smokeSuiteMinimalTestVersion      = "1.19.0"
+	objectSuiteMinimalTestVersion     = "1.19.0"
```

Comment:
These versions were only used for jenkins tests, so we should be able to clean up the code that was using them (in a separate PR)

Reply:
Yeah. I think we need it for now since it's used in the call to `StartTestCluster()`

Reply:
Did this really quickly for a follow-up: https://github.com/rook/rook/pull/8736/commits/5cbf50b10d0174278c44fa85bc5ea68afd864994